src/anurag/Assignment22.java

```diff
@@ -0,0 +1,29 @@
+//Print all prime numbers between 2 to 100
+
+package anurag;
+
+public class Assignment22 {
+
+	void display() {
+		for (int i = 2; i < 100; i++) {
+			{
+				int flag = 0;
+				for (int j = 2; j <= i - 1; j++) {
+					if (i % j == 0)
+						flag = 1;
```

Comment:
once you get number is not prime then you should break the loop, it will reduce program time execution